data/original/carrier-services.csv

```diff
@@ -45,6 +45,7 @@ ilg-express-tracked,Express Tracked,ilg
 ilg-global-courier,Global Courier,ilg
 ilg-international-courier,International Courier,ilg
 ilg-standard-tracked,Standard Tracked,ilg
+malca-amit,Malca Amit,malca
```

Comment:
there should be two services here:
- armored
- express

based on the integration docs in https://my.malca-amit.com/integration/api

<img width="590" alt="Screen Shot 2022-01-25 at 2 57 20 PM" src="https://user-images.githubusercontent.com/85070/151049995-08a33819-a9e0-499f-9b93-dbd6148a25e5.png">

Reply:
ack, thanks!